src/Billing/Controllers/StripeController.cs

```diff
@@ -164,6 +167,13 @@ public async Task<IActionResult> PostWebhook([FromQuery] string key)
                 // org
                 if (ids.Item1.HasValue)
                 {
+                    // sponsored org
+                    if (IsSponsoredSubscription(subscription))
+                    {
+                        await _organizationSponsorshipService
+                            .ValidateSponsorshipAsync(ids.Item1.Value);
+                    }
```

Comment:
This would make a better method than comment, I think. `CheckForSponsership()` or something like that.